src/main/java/com/napier/sem/models/reports/LanguageReportRow.java

```diff
@@ -0,0 +1,9 @@
+//code by viva
+
+package com.napier.sem.models.reports;
+
+public class LanguageReportRow {
+	public String language;
+	public double speakers;

```

Comment:
Speakers should be an integer since you cannot count 0.5 people. Replace double with integer.